Src/DependencyCollector/NuGet/ApplicationInsights.config.install.xdt

```diff
@@ -7,6 +7,7 @@
       <ExcludeComponentCorrelationHttpHeadersOnDomains>
         <!-- 
         Requests to the following hostnames will not be modified by adding correlation headers. 
+        This is only applicable if Profiler is installed - StatusMonitor or Azure Extension.
```

Comment:
nitpick: grammar is unclear, does it mean "...is installed via StatusMonitor..."?